docs/docs/api-reference/class.md

```diff
@@ -110,6 +110,53 @@ Use caution when setting resource limits. Setting limits that are too low may
 result in the `Piscina` worker threads being unusable.
 :::
 
+## `PiscinaHistogram`
+
+The `PiscinaHistogram` allows you to access the histogram data for the pool of worker threads.
```

Comment:
Can we have a code sample on how users would do it in userland?

Reply:
Done, let me know if that's enough or if there's a something else to cover